### PR TITLE
Kth index removal from a string

### LINKED ISSUE #419: To remove the kth index from a string .
A program that will accept a string and if any character is present at kth index then it will remove that character and print the rest string.If the index given is more than the length then it will return the complete string.

Please assign me this issue. @rcoder23

### PR DESCRIPTION
**Pull request by Khushi-agarwal**
Fixes #419

### CODE REVIEW

**Comment by Khushi-agarwal** [comment]
Please review it. @rcoder23 @rashrey2308 .Thanks

File: Dynamic Programming/Kth index in string.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ 	#Check whether the value of k is greater than length
+     if(k>l):
+         print(s)
+     #If k is less than length of string then remove the kth index value
+     else:
+          s1=''
+          for i in range(0,l):
+              if(i!=k):
+                  s1=s1+s[i]
+          print(s1)
```
Comment (rcoder23) [inline Dynamic Programming/Kth index in string.py:20]: extra indention

**Review by rcoder23 — CHANGES_REQUESTED** [review]
recheck code

**Comment by Khushi-agarwal** [comment]
@rcoder23 done the changes.Kindly review.

File: Dynamic Programming/Kth index in string.py (hunk 1 above)
Comment (Khushi-agarwal) [reply Dynamic Programming/Kth index in string.py]: Done the changes.

**Review by rcoder23 — CHANGES_REQUESTED** [review]
Add solution in string folder

**Comment by Khushi-agarwal** [comment]
@rcoder23 added to string folder

**Comment by Khushi-agarwal** [comment]
@rashrey2308 @Joshitha18 kindly review my pr and merge it.Thanks.

**Comment by rcoder23** [comment]
@Khushi-agarwal Kth index in string.py rename file as Kth_index_remove.py

**Comment by Khushi-agarwal** [comment]
> @Khushi-agarwal Kth index in string.py rename file as Kth_index_remove.py

Done.Kindly review.

**Comment by Khushi-agarwal** [comment]
@rcoder23 please merge it if everything is right.Thanks.

**Review by rcoder23 — APPROVED** [review]

**Pull request merged**